qiita_db/user.py

```diff
@@ -232,10 +233,15 @@ def create(cls, email, password, info=None):
         # create user default sample holder
         sql = ("INSERT INTO qiita.analysis "
                "(email, name, description, dflt, analysis_status_id) "
-               "VALUES (%s, %s, %s, %s, 1)")
+               "VALUES (%s, %s, %s, %s, 1) RETURNING analysis_id")
         conn_handler.add_to_queue(queue, sql,
                                   (email, '%s-dflt' % email, 'dflt', True))
 
+        portal_id = convert_to_id('QIITA', 'portal_type', 'portal')
```

Comment:
Just a general comment. Note that you are doing partial changes here. I would be more happy if you actually do all the changes rather than just half of it. This way of working is really dangerous as there is a high probability that this slips through... but that's why I made the note on the WIP PR.

Also, it is better if you comment that this is going to be solved in another PR rather than just ignoring the comment...